protocol/scripts/affiliates/update_affiliate_parameters.py

```
@@ -0,0 +1,177 @@
+                "submit-proposal",
+                tmp_file_path,
+                "--from=alice",
+                "--gas=auto", 
+                "--fees=10000000000000000000000adv4tnt",
+                "--node=" + args.node,
+                "--chain-id=" + args.chain_id,
+                "--keyring-backend=test", 
+                "--yes"
+            ]
+            
+            result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)